lightning-c-bindings/demo.cpp

```diff
@@ -115,18 +115,21 @@ static int num_txs_broadcasted = 0; // Technically a race, but ints are atomic o
 void broadcast_tx(const void *this_arg, LDKTransaction tx) {
 	num_txs_broadcasted += 1;
 	//TODO
+	Transaction_free(tx);
 }
 
 struct NodeMonitors {
 	std::mutex mut;
 	std::vector<std::pair<LDK::OutPoint, LDK::ChannelMonitor>> mons;
-	LDKChainWatchInterfaceUtil* watch_util;
 	LDKLogger* logger;
+
+	void ConnectBlock(const uint8_t (*header)[80], uint32_t height, LDKCVec_C2Tuple_usizeTransactionZZ tx_data, LDKBroadcasterInterface broadcast, LDKFeeEstimator fee_est) {
```

Comment:
Hmm, it was just the path of least resistance to adapt the existing code to the new API. I kinda like this cause its just *more* code, which covers more cases for the bindings, even if it covers less actual Rust code. It'd be great to have more than the two demos so that we could use different features.